site/en/examples/chat_calculator.ipynb

```diff
@@ -11,7 +11,7 @@
     },
```

Comment:
<p><span style="color: rgba(0, 0, 0, 0.87); letter-spacing: 0.01em;">These steps mention </span><code style="font-weight: 400; letter-spacing: 0.01em;">generate_text</code><span style="color: rgba(0, 0, 0, 0.87); letter-spacing: 0.01em;">, but this is the </span><code style="font-weight: 400; letter-spacing: 0.01em;">chat</code><span style="color: rgba(0, 0, 0, 0.87); letter-spacing: 0.01em;"> notebook.</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
thanks for catching that! fixed in the next commit